pkg/cmd/grafana-cli/utils/grafana_path.go

```diff
@@ -6,35 +6,51 @@ import (
 	"path/filepath"
```

Comment:
Some fixes here because after last change, you could run grafana-cli from repo root, it would install the plugin to wrong directory.